modules/kubernetes-cluster/variables.tf

```diff
@@ -38,7 +38,7 @@ variable "agents_size" {
 
 variable "kubernetes_version" {
   description = "Version of Kubernetes to install"
-  default     = "1.11.3"
+  default     = "1.14.8"
```

Comment:
This works as an interim fix but I'd like to see it using a data source to pull the latest stable version of k8s for the region